### PR TITLE
More flexibility with multitouch gesture events

### PR DESCRIPTION
**Pull request by SirKnightDragoon**
Give the possibility to get the local position with your own global position. 
Save the global position in the changedTouches event object.
This feature give us the possibility to manage multitouch event in the save object ex: Resizing Gesture event with two touch moving in the same time.

### CODE REVIEW

**Comment by SirKnightDragoon** [comment]
### Exemple (zoom gesture):

```
onTouchMove = function(e){
	if(e.originalEvent.touches != undefined){
		if(e.originalEvent.touches.length == 1){
			this.state = "move";
			if(e.originalEvent.changedTouches[0].identifier == this.firstEndPosition.id){
				e.getLocalPosition(this, this.firstEndPosition.position);
			}else if(e.originalEvent.changedTouches[0].identifier == this.secondEndPosition.id){
				e.getLocalPosition(this, this.secondEndPosition.position);
			}
		}else if(e.originalEvent.touches.length == 2){
			this.state = "changezoom";

			if(e.originalEvent.changedTouches[0].identifier == this.firstEndPosition.id){
				e.getLocalPosition(this, this.firstEndPosition.position, {x:e.originalEvent.changedTouches[0].globalX, y:e.originalEvent.changedTouches[0].globalY});
			}else if(e.originalEvent.changedTouches[0].identifier == this.secondEndPosition.id){
				e.getLocalPosition(this, this.secondEndPosition.position, {x:e.originalEvent.changedTouches[0].globalX, y:e.originalEvent.changedTouches[0].globalY});
			}
			if(e.originalEvent.changedTouches.length == 2){
				if(e.originalEvent.changedTouches[1].identifier == this.firstEndPosition.id){
					e.getLocalPosition(this, this.firstEndPosition.position, {x:e.originalEvent.changedTouches[1].globalX, y:e.originalEvent.changedTouches[1].globalY});
				}else if(e.originalEvent.changedTouches[1].identifier == this.secondEndPosition.id){
					e.getLocalPosition(this, this.secondEndPosition.position, {x:e.originalEvent.changedTouches[1].globalX, y:e.originalEvent.changedTouches[1].globalY});
				}
			}
		}
	}else{
		this.state = "move";
	}

	console.log(this.state + " Touch 1 ("+this.firstEndPosition.id+"): " + this.firstEndPosition.position.x + " x " + this.firstEndPosition.position.y + " Touch 2: ("+this.secondEndPosition.id+")" + this.secondEndPosition.position.x + " x " + this.secondEndPosition.position.y);
}
```

**Comment by englercj** [comment]
Please don't include build files in the PR.

**Comment by SirKnightDragoon** [comment]
Now is it good ?

File: src/pixi/InteractionData.js [hunk 1]
```diff
@@ -43,10 +43,10 @@ PIXI.InteractionData = function()
  * @param [point] {Point} A Point object in which to store the value, optional (otherwise will create a new point)
  * @return {Point} A point containing the coordinates of the InteractionData position relative to the DisplayObject
  */
-PIXI.InteractionData.prototype.getLocalPosition = function(displayObject, point)
+PIXI.InteractionData.prototype.getLocalPosition = function(displayObject, point, globalPos)
 {
     var worldTransform = displayObject.worldTransform;
-    var global = this.global;
+    var global = globalPos ? globalPos : this.global;
```
Comment (adelciotto) [inline src/pixi/InteractionData.js]: you should check for optional params like so:

```javascript
var global = (globalPos === 'undefined') ? this.global : globalPos;
```

The logic fails if globalPos is passed, but evaluates as false.

File: src/pixi/InteractionData.js [hunk 2]
```diff
@@ -43,10 +43,10 @@ PIXI.InteractionData = function()
  * @param [point] {Point} A Point object in which to store the value, optional (otherwise will create a new point)
  * @return {Point} A point containing the coordinates of the InteractionData position relative to the DisplayObject
```
Comment (adelciotto) [inline src/pixi/InteractionData.js]: Add some documentation about the new optional parameter.

Comment (englercj) [inline src/pixi/InteractionData.js]: :+1:

**Comment by GoodBoyDigital** [comment]
Lovely PR @SirKnightDragoon! If you could tweak the docs will role into v3 today. Thanks very much :+1:

**Comment by SirKnightDragoon** [comment]
Done :)

**Comment by GoodBoyDigital** [comment]
slliiiiccck! :+1:

**Pull request merged**